core/domain/prod_validators.py

```diff
@@ -28,8 +28,6 @@
 from core.domain import exp_services
 from core.domain import rights_domain
 from core.domain import rights_manager
-from core.domain import subtopic_page_domain
-from core.domain import subtopic_page_services
 from core.platform import models
 import feconf
 import python_utils
```

Comment:
Remove `subtopic_models` from line 39*

Reply:
Hi, can you please check the file again?
All the changes required to this file were made before @DubeySandeep

Reply:
I checked the [file](https://github.com/oppia/oppia/blob/8df803bb9a096fc72ef6e84564294f04d88e7574/core/domain/prod_validators.py) again and I think `subtopic_models` defined on line 39 is no more needed, can you please check and remove (if required)?

Reply:
Done!